### PR TITLE
Fix collapse work on icon and empty area only

### LINKED ISSUE #2204: Reduce surface area of docstring folding
I often click on docstring headers in order to copy the URL pointing to it. This folds the docstring which is a bit annoying. Would it be possible to reduce the surface area to fold? Perhaps limited to the arrow, at least if the docstring is open.

### PR DESCRIPTION
**Pull request by Hetarth02**
Fixes #2204.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -24,6 +24,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Fixed
 
 * The search prompt in the HTML output again correctly handles parenthesis and other special character that would previously cause the search to crash. ([#2513])
+* Collapse will work when clicked on icon and empty area only. ([#2204], [#2551])
```
Comment (mortenpi) [inline CHANGELOG.md:27]:
```suggestion
* Collapsing of docstrings in the HTML output can now only be triggered when clicking on the icon or the empty area only. ([#2204], [#2551])
```

**Comment by Hetarth02** [comment]
>spacing around the &mdash is a bit uneven now

@mortenpi Fixed, should render same as before now.

**Review by mortenpi — APPROVED** [review]
LGTM, thanks!

**Comment by Hetarth02** [comment]
@mortenpi You might need to merge this manually or enable auto-merge again.

I think my recent push disabled auto-merge.

**Pull request merged**